pyproject.toml

```
@@ -32,7 +32,7 @@ include = ["strawberry_django_plus/py.typed"]
 [tool.poetry.dependencies]
 python = "^3.8"
 django = ">= 3.2"
-strawberry-graphql = ">= 0.140.3"
+strawberry-graphql = ">=0.140.3"
```

This is what is read by pip when installing the package. We need to bump it as well since after this release, it will not be compatible with older versions anymore

```suggestion
strawberry-graphql = ">=0.160.0"
```

---

Thanks for taking a look! There was a `0.161.0` release a few hours ago, so I've updated the version number here as well.